packages/views/accounts_header_buttons.jade

```diff
@@ -2,6 +2,9 @@ template(name="accountsHeaderButtons")
   ul.nav.navbar-nav.navbar-right
     if currentUser
       li
+        a.navbar-btn(href="{{path route='organizations'}}")
+          span Organizations
```

Comment:
I'm not sure this belongs in the accounts header buttons as opposed to just the header.